README.md

```diff
@@ -1,34 +1,43 @@
 ![](./resources/official_armmbed_example_badge.png)
 # Running Mbed Crypto examples on Mbed OS
-This repository contains a set of examples demonstrating the compilation and use of Mbed Crypto on Mbed OS.
-
-List of examples contained within this repository:
-* Example code snippets for using the library, with [documentation](https://github.com/ARMmbed/mbed-crypto/blob/development/docs/getting_started.md).
+This repository contains an example demonstrating the compilation and use of Mbed Crypto on Mbed OS.
 
 ## Prerequisites
-* Install <a href='https://github.com/ARMmbed/mbed-cli#installing-mbed-cli'>Mbed CLI</a>
+This example enables Mbed OS PSA. It is compatible with **non-TF-M** targets, as TF-M has its own PSA implementation
```

Comment:
This example should work with TF-M. If not, we need to understand why it doesn't and fix it.

Reply:
I've updated it. Hopefully we can get this PR in to unblock Travis, and #95 actually fixes TF-M support.